addons/udes_priorities/models/stock_picking.py

```diff
@@ -1,3 +1,4 @@
+import ipdb
```

Comment:
Adding more random imports again? :smile:

Reply:
Also need to make the commit message under 80 characters

Reply:
Done :)